examples/utils/generic.py

```diff
@@ -7,10 +7,51 @@
 import urllib
 from IPython.display import display, JSON
 import json
+from tabulate import tabulate
 
 
 # GENERIC UTILITIES
 
+def update_settings_json_with_additional_variables(notebook_environment="Jupyter", **kwargs):
```

Comment:
A generic util should be generic - let's just have `update_json_file_kwargs` maybe? And pass the path to `settings.json` to it upon invocation?

Reply:
Hi Timur, do you mean something like this?

update_json_file_kwargs(path_to_json_file, notebook_environment="Jupyter", **kwargs):
    <update json variables in here if change detected>

Where we have now passed the path the settings.json to the util function?

Reply:
Yes, or `update_json_file_from_kwargs`